src/pages/ProductList/ProductList.js

```diff
@@ -13,19 +13,57 @@ class ProductList extends Component {
       productFits: '',
       filterMoveNum: 1,
       grayDisplayNum: -1,
+      filterPrice: '',
+      filterColor: [],
+      filterFit: [],
     };
   }
 
   componentDidMount() {
-    fetch('./data/productListData-doyoung.json')
+    fetch('http://10.58.2.246:8000/products?limit=28&offset=0')
```

Comment:
api 주소는 자주 변경되고, 여러 군데에서 사용될 수 있기 때문에 src/config.js 같은 경로에 분리하여 사용해주세요
```suggestion
    fetch(`${API}/products?limit=28&offset=0`)
```

Reply:
🙆‍♂️수정완료. comfig로 분리하였습니다.